specs/WinRT/WinRTAPIContracts.md

```diff
@@ -0,0 +1,88 @@
+# New Document
+# Windows App SDK API Contracts
+
+## Summary
+Customers trust that Microsoft products are sound. As Windows App SDK evolves and more developers
+use Windows App SDK in their own projects Windows App SDK needs a way, at build time and run time,
+to allow apps to light up API features and to gracefully degrade. This can be achieved by looking
+at the deltas between APIs.
+
+In order to allow apps to depend on API versions Windows App SDK will add contracts to all their
+WinRT APIs.
+
+## Architecture Decisions
+1. Contract boundaries
+   1. One contract per feature/component/sub-system.
+   2. Contracts will not span across a transport package/repo.
+2. Contract numbering  
+Contracts versions are one number, starting with 1 and is incremented for any interface
```

Comment:
This could use some clarification. A reader new to the topic might think every single change within a release requires a new contract version. Perhaps something like this:

> Contract version numbers are a single number, with no minor "x.y" versions. The first version of a contract will be version 1. Contract versions are frozen upon a stable release of Windows App SDK, any changes within a contract after that stable release will require incrementing the version number.